sql/core/src/main/scala/org/apache/spark/sql/execution/objects.scala

```diff
@@ -389,6 +389,102 @@ case class AppendColumnsWithObjectExec(
     copy(child = newChild)
 }
 
+/**
+ * Groups the input rows together and calls the function with each group and an iterator containing
+ * all elements in the group.  The result of this function is flattened before being output. This
+ * version of the Physical operator takes a user provided initial state.
```

Comment:
param docs

Reply:
improve the docs as well. "This physical operator is similar to MapGroupsExec but the difference that it calls the user-defined function with .... group + iterator + user-defined initial state"